.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  workflow_call: {}
+
+name: Build
+
+permissions:
+  contents: write
+  pull-requests: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "nothing to do."
+        shell: bash
```

Comment:
What's the purpose of this?

Reply:
This is what I was mentioning yesterday - because our native packages require a build step, it's difficult to share a generation script that works for both without basically programmatically generating yml.  which is an option in the future.

the easiest way to share the code was to require all packages to have a build phase, but for non-native packages it's a no-op.